google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminSettingsTest.java

```diff
@@ -158,14 +164,16 @@ public void testToString() throws IOException {
     assertThat(defaultSettings.toString()).doesNotContain("totalTimeout=PT13H32M");
     assertThat(settings.toString()).contains("totalTimeout=PT13H32M");
 
-    int nonStaticFields = 0;
+    List<String> nonStaticFields = new ArrayList<>();
     for (Field field : BigtableTableAdminStubSettings.class.getDeclaredFields()) {
       if (!Modifier.isStatic(field.getModifiers())) {
-        nonStaticFields++;
+        nonStaticFields.add(field.getName());
       }
     }
     // failure will signal about adding a new settings property
-    assertThat(SETTINGS_LIST.length).isEqualTo(nonStaticFields);
+    List<String> sortedSettingsList = Arrays.stream(SETTINGS_LIST).sorted().collect(Collectors.toList());
+    List<String> sortedNonStaticFields = nonStaticFields.stream().sorted().collect(Collectors.toList());
+    assertThat(sortedSettingsList).isEqualTo(sortedNonStaticFields);
```

Comment:
I think this would be cleaner
```java
List<String> declaredFieldNames = Arrays.stream(BigtableTableAdminStubSettings.class.getDeclaredFields())
        .filter(f -> Modifier.isStatic(f.getModifiers()))
        .map(Field::toString)
        .collect(Collectors.toList());

    assertThat(declaredFieldNames).containsExactlyElementsIn(SETTINGS_LIST);
```